filters/auth/webhook_test.go

```diff
@@ -15,15 +15,17 @@ import (
 )
 
 const headerToCopy = "X-Copy-Header"
+const webhookRedirectLocation = "https://example.com/auth"
```

Comment:
Please use: `"https://redirect.example/auth"` as by RFC this is in `.example`, so better than in `.com`.